docs/src/theme/CodeBlock/index.js

```diff
@@ -203,9 +203,6 @@ export default function CodeBlock({
             className={clsx(
               styles.codeBlockContent,
               language,
-              !hideButton
-                ? styles.userSelectionAllowed
-                : styles.userSelectionDisabled
```

Comment:
why are we removing this?

Reply:
`pointer-events: none;
  user-select: none;` 
  -----
  
  it was preventing horizontal scroll in codeblock